docs/Lua API.rst

```diff
@@ -1073,6 +1089,14 @@ Announcements
   Uses the type to look up options from announcements.txt, and calls the above
   operations accordingly. The units are used to call ``addCombatReportAuto``.
 
+* ``dfhack.gui.autoDFAnnouncement(report,text[,log_failures])``
+  ``dfhack.gui.autoDFAnnouncement(type,pos,text,color[,is_bright,unit1,unit2,is_sparring,log_failures])``
+
+  Takes a ``df.report_init`` and a string and processes them just like DF does.
+  Sometimes this means the announcement won't occur. Set ``log_failures`` to ``true`` to
+  log the reason why to the dfhack console (e.g., unrevealed map or wrong gamemode.)
+  Can also be built from parameters instead of a ``report_init``.
```

Comment:
Can we assume that if someone is calling this, they know what the types and purpose of all the parameters are?